src/main/java/de/retest/recheck/ui/diff/RootElementDifferenceFinder.java

```diff
@@ -18,6 +21,11 @@
 
 	private final ElementDifferenceFinder elementDifferenceFinder;
 
+	private final Double minimumWindowMatch =
+			getConfiguredDouble( WINDOW_MATCH_THRESHOLD_PROPERTY, WINDOW_MATCH_THRESHOLD_DEFAULT );
+	private final Double minimumContainedComponentsMatchMatch = getConfiguredDouble(
```

Comment:
I would inline this so that it is evaluated every time it is needed (in case it is not set at construction of this class). Also this is only used once.

Reply:
I don't think this is necessary.